2200001/문제5.py

```diff
@@ -0,0 +1,23 @@
+# 신용카드 만들기 1
+T = int(input())
+
+for t in range(1, T+1):
+    card = list(map(int, input().split()))
+    num = 0
+    num1 = 0
+
+    for i in range(len(card)):
+        if (i+1) % 2 == 1:
+            num += int(card[i]) * 2
+        else:
+            num1 += int(card[i])
```

Comment:
좋은 풀이네요 👍 
```suggestion
    for i in range(len(card)):
        if (i+1) % 2 == 1:
            num += card[i] * 2
        else:
            num1 += card[i]
```
이미 `card`의 모든 원소들이 integer이므로, 다시 `int` 함수를 사용하지 않으셔도 괜찮습니다 😄